services/movie_session.py

```
@@ -24,6 +24,15 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
     return MovieSession.objects.get(id=movie_session_id)
 
 
+def get_taken_seats(session_id: int) -> list:
+    session = get_movie_session_by_id(session_id)
```

The `get_movie_session_by_id` function uses `objects.get()`, which will raise a `DoesNotExist` exception if a session with the given ID is not found. This exception is not handled here, which means your program will crash. The `if not session:` check on the next line is unreachable. To fix this, you should wrap this call in a `try...except MovieSession.DoesNotExist:` block to handle this case and return an empty list as intended.